doc/rtd/topics/module_creation.rst

```diff
@@ -70,6 +70,10 @@ Guidelines
     module definition in ``/etc/cloud/cloud.cfg[.d]`` has been modified
     to pass arguments to this module.
 
+* Your module name must be included in `cloud-init-schema.json`_
```

Comment:
I think you meant this file, right? `cloudinit/config/schemas/schema-cloud-config-v1.json`

Reply:
Yeah, updated.